fe/src/components/commons/Icon/IconStyle.ts

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+import { IconStyleProps } from './Icon';
+
+export const Icon = styled.div<IconStyleProps>`
+  font-family: 'SF Pro', fallback, sans-serif;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: ${({ size }) => size}px;
+  height: ${({ size }) => size}px;
+  font-size: ${({ size }) => size}px;
+  color: ${({ theme, color }) =>
+    color ? theme.colors[color] : theme.colors.neutralText};
```

Comment:
[적극]
앞서 size는 기본값으로 16을 내려주었고, 그것이 합당하다고는 생각했는데요,
color는 스타일드 컴포넌트에서 undefined 검사를 해서 바꿔주고 있네요. 
사용하는 차원에서는 어떤것은 기본값이 드러나있고, 어떤 것은 드러나있지 않아서 얼라인이 맞지 않다는 느낌을 받습니다.

[제안]
개인적으로는 이 경우도 기본값을 할당해주는게 좋지 않은가 해요.
다만, theme으로 쓰는 지라 컴포넌트의 기본값으로서는 활용이 어려울 수 있겠군요. (이런 부분이 전역테마를 프로바이딩하는 것의 불편한점이라 생각해요.)

Reply:
그렇게 변경하도록 하겠습니다!

Reply:
아 이 부분은 타입스크립트 때문에 이렇게 해야만 하는 상황입니다.
그러니까
1. color 와 size의 기본 값을 prop을 정의할 때부터 지정해두었음. 이렇게 하면 외부에서 color size을 지정하지 않아도 됨
<img width="1053" alt="Screenshot 2023-06-12 at 3 12 26 PM" src="https://github.com/codesquad-members-2023/second-hand/assets/96381221/978b01e9-fa68-420c-9d36-38392fa2d6c7">
2. 그러나 이렇게 되면 이런 타입 에러가 발생합니다. 그래서 이곳에서도 기본 값을 지정해두긴 해야 하는 상황입니다.

Reply:
`import { css } from 'styled-components'`
```suggestion
${({ theme, color }) => color && css`color: theme.colors[color]`;}
```
IDE에서 체크해보지는 않았는데 요러면 되지 않을까요?